modules/osdk-run-proxy.adoc

```diff
@@ -42,7 +42,7 @@ This tutorial uses `HTTP_PROXY` as an example environment variable.
 .Procedure
 ifdef::golang[]
 . Edit the `controllers/memcached_controller.go` file to include the following:
-.. Import the `proxy` package from the link:https://github.com/operator-framework/operator-lib/releases/tag/v0.7.0[`operator-lib`] library:
+.. Import the `proxy` package from the link:https://github.com/operator-framework/operator-lib[`operator-lib`] library:
```

Comment:
Note that this is a link to the actual repo for the purposes of a developer's Operator project, not a link for upstream documentation.

Reply:
Thank you for the note @adellape